Tools/Scripts/build-and-collect-pgo-profiles

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+set -e
+set -o pipefail
+
+# Build WebKit, run benchmarks, and spit out compressed PGO profiles
+BASE=/Volumes/WebKit/BenchmarkProfiles/
```

Comment:
Should we make this a positional arguments and defaults to `/Volumes/WebKit/BenchmarkProfiles/` for better usability?